gui/default/assets/lang/lang-en.json

```diff
@@ -151,6 +155,8 @@
    "Introduced By": "Introduced By",
    "Introducer": "Introducer",
    "Inversion of the given condition (i.e. do not exclude)": "Inversion of the given condition (i.e. do not exclude)",
+   "Items have been added to this folder which are not encrypted.": "Items have been added to this folder which are not encrypted.",
+   "Items in Receive Encrypted": "Items in Receive Encrypted",
```

Comment:
Not sure what this means. Is 'Receive' a name of a place?

Reply:
"Receive Encrypted" is a folder type. Anyways, clearly a terrible label. It means "Not encrypted/locally changed items in a receive-only folder (where nothing should ever change locally)". So we might as well use the same label as for receive-only folders: "Locally Changed Items".